contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -8,749 +8,24 @@ doc: |
   
   .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
   .. _IUPAC Recommendations 2020: https://doi.org/10.1515/pac-2019-0404
-symbols:
-  doc: |
-    The symbols used in the schema to specify e.g. dimensions of arrays
-  n_transmission_function: |
-    Number of data points in the transmission function.
 type: group
-NXmpes(NXobject):
+NXmpes(NXphotoemission):
   (NXentry):
-    definition:
-      \@version:
-      enumeration: [NXmpes]
-    title:
-    start_time(NX_DATE_TIME):
-      doc: |
-        Datetime of the start of the measurement.
-        Should be a ISO8601 date/time stamp. It is recommended to add an explicit time zone,
-        otherwise the local time zone is assumed per ISO8601.
-    end_time(NX_DATE_TIME):
-      exists: recommended
-      doc: |
-        Datetime of the end of the measurement.
-        Should be a ISO8601 date/time stamp. It is recommended to add an explicit time zone,
-        otherwise the local time zone is assumed per ISO8601.
-    method:
-      exists: recommended
-      doc: |
-        Name of the experimental method.
-        
-        If applicable, this name should match the terms given by `Clause 11`_ of
-        the `ISO 18115-1:2023`_ specification.
-        
-        Examples include:
-            * X-ray photoelectron spectroscopy (XPS)
-            * angle-resolved X-ray photoelectron spectroscopy (ARXPS)
-            * ultraviolet photoelectron spectroscopy (UPS)
-            * angle-resolved photoelectron spectroscopy (ARPES)
-            * hard X-ray photoemission spectroscopy (HAXPES)
-            * near ambient pressure X-ray photoelectron spectroscopy (NAPXPS)
-            * photoelectron emission microscopy (PEEM)
-            * electron spectroscopy for chemical analysis (ESCA)
-            * time-resolved angle-resolved X-ray photoelectron spectroscopy (trARPES)
-            * spin-resolved angle-resolved X-ray photoelectron spectroscopy (spin-ARPES)
-            * momentum microscopy
-        
-        .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
-        .. _Clause 11: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:sec:11
```

Comment:
Should this maybe also be specialized from NXmpes (e.g. exclude PEEM without energy axis)?

Reply:
I would say it is fine for now and we can just keep a generic list of examples in NXphotoemission and then we can see how the hierarchy evolves with time and which techniques are collected with which definition.